index.css

```diff
@@ -29,33 +41,82 @@ p {
   line-height: 52px;
 }
 
+.header-3 {
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.body-1 {
+  font-size: 20px;
+  line-height: 24px;
+}
+
 .body-2 {
   font-size: 16px;
   line-height: 24px;
   font-weight: normal;
 }
 
+.body-3 {
+  font-size: 15px;
+  line-height: 24px;
+}
+
 .bolder {
   font-weight: 500;
 }
 
+.bolder-2 {
+  font-weight: 600;
+}
+
 .drop-shadow {
   filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
 }
 
+.caption {
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.small {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
 /* Colors */
 .color-b300 {
   color: #7f8cff;
 }
 
+.color-b400 {
+  color: #6070ff;
+}
+
+.color-n100 {
+  color: #7a869a;
+}
+
 .color-n600 {
   color: #344563;
 }
 
+.color-n700 {
+  color: #253858;
+}
+
 .color-n800 {
   color: #172b4d;
 }
 
+.color-black {
+  color: #000;
+}
+
```

Comment:
Try using descriptive names with the classes as it will make it easy to find later on